packages/mask/src/extension/popups/render.tsx

```diff
@@ -27,6 +27,8 @@ startPluginDashboard(
             personaSign: Services.Identity.signWithPersona,
             walletSign: Services.Ethereum.personalSign,
             currentPersona: currentPersonaSub,
+            silentSign: Services.Identity.generateSignResult,
```

Comment:
You did not do the permission check. Please only allow this on the plugin you want to.